html-aam/index.html

```diff
@@ -4599,7 +4603,9 @@ <h4 id="el-label">`label`</h4>
             <!-- <th><a href="#accessible-name-and-description-computation">Naming Algorithm</a></th> -->
             <tr>
               <th>Comments</th>
-              <td></td>
+              <td>
+                <p class="note">If a `label` element is hidden from the accessibility tree, then it will not provide an accessible name to the labelable element is is associated with.</p>
```

Comment:
```suggestion
                <p class="note">If a `label` element is hidden from the accessibility tree, then it will not provide an accessible name to the <a data-cite="html//forms.html#category-label">labelable element</a> it is associated with.</p>
```